SpatialGDK/Source/SpatialGDK/Private/SpatialGDKSettings.cpp

```diff
@@ -57,6 +59,12 @@ void USpatialGDKSettings::PostEditChangeProperty(FPropertyChangedEvent& Property
 			FModuleManager::LoadModuleChecked<ISettingsModule>("Settings").ShowViewer("Project", "SpatialGDKEditor", "Editor Settings");
 		}
 	}
+
+	if (PropertyName == GET_MEMBER_NAME_CHECKED(USpatialGDKSettings, bUsingOffloading))
+	{
+		GetMutableDefault<ULevelEditorPlaySettings>()->SetOffloadingEnabled(bUsingOffloading);
```

Comment:
I think here you may need to trigger a `PostEditChange` here for the `ULevelEditorPlaySettings` - but not sure if entirely necessary

Reply:
Why, it seems to be working for me right now, maybe I'm missing a test case?

Reply:
I guess in the future if someone wants to add an edit trigger for this property it may not fire appropriately, also unsure what happens in saving/loading cases